src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -262,9 +265,8 @@ def _construct_job_output_state_set(
                 else:
                     # fetch the current state without updating it
                     output_states[job_id] = job.output_state({})
-                    # set the status to 'ee2_error' and the 'updated' timestamp to now
-                    output_states[job_id]["jobState"]["status"] = "ee2_error"
-                    output_states[job_id]["jobState"]["updated"] = int(time.time())
+                    # add an error field with the error message from the failed look up
+                    output_states[job_id]["error"] = error_message
```

Comment:
change the handling of ee2 errors so that now the error is reported in a separate `error` field in the response object, rather than changing the job state data.